app/controllers/projects_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
                        .order('last_github_update DESC NULLS LAST')
                        .order('commit_count DESC NULLS LAST')
                        .includes(:user)
-    @projects_languages_array = Language.all.collect(&:name)
+    @projects_languages_array = Language.all.collect(&:name).uniq
```

Comment:
Here you could do `Language.pluck(:name).uniq` 👍

Reply:
hey @nisevi, Thanks for the tip!! :smiley: